doc/examples/xx_applications/plot_face_detection.py

```diff
@@ -0,0 +1,106 @@
+"""
+=========================================
+Face detection using a cascade classifier
+=========================================
+
+This example shows how to detect faces on an image using object detection
+framework.
```

Comment:
Maybe add a sentence like "It is an illustration of how to use machine learning for computer vision" ?